.github/workflows/tlaplus.yml

```diff
@@ -0,0 +1,15 @@
+name: "TLA+ Spec Testing"
+
+on: [push, pull_request]
```

Comment:
Can we filter and run this only on pull requests that touch a *.tla file, or this .yml file? The vast majority of pushes or pull requests will not contain a relevant change.

Reply:
Using https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths, probably.

Reply:
You would definitely want to include changes to `*.cfg` as well.   However, it is probably best to trigger a build on any change to the `tla/` folder.

Reply:
Could we also enable on `workflow_dispatch` so we have a button to run on demand (for instance, if there's a new release of tlc/community modules and we want to see the change of checking time).

I have implemented a TLA+ CI here if that's any help: https://github.com/fpaxos/fpaxos-tlaplus/blob/main/.github/workflows/tlatools.yml

Reply:
Another inspiration (doesn't use https://github.com/pmer/tla-bin.git, which IIRC doesn't pass JVM properties):

https://github.com/tlaplus-workshops/ewd998/blob/main/.github/workflows/main.yml